cmd/vic-machine/create/create.go

```diff
@@ -1440,7 +1440,7 @@ func (c *Create) Run(clic *cli.Context) (err error) {
 		}
 	}()
 
-	if err = executor.CheckServiceReady(ctx, vchConfig, c.clientCert); err != nil {
+	logErr := func() {
```

Comment:
This is an arguable change. I would not make a closure here. keeping this logic where it was is ok.